### DISCUSSION TITLE
Error in documentation

### DISCUSSION THREAD

**Discussion by Danne-J** [blakeblackshear/frigate#6012] [General]
https://docs.frigate.video/configuration/detectors

same in dev branch
https://github.com/blakeblackshear/frigate/blob/dev/docs/docs/configuration/detectors.md

**Nvidia**
Generate Models
wget https://raw.githubusercontent.com/blakeblackshear/frigate/docker/tensorrt_models.sh

gives 404

https://raw.githubusercontent.com/blakeblackshear/frigate/dev/docker/tensorrt_models.sh
or
https://github.com/blakeblackshear/frigate/blob/v0.12.0/docker/tensorrt_models.sh
is working.

trying to create a plane issue but was not able to find any template that adressed this kind of issue

**Comment by NickM-27** [comment]
Thanks, addressed in https://github.com/blakeblackshear/frigate/pull/6015
